server/service/apple_mdm.go

```diff
@@ -2978,6 +2978,8 @@ func (svc *MDMAppleDDMService) DeclarativeManagement(r *mdm.Request, dm *mdm.Dec
 		return nil, ctxerr.New(r.Context, "missing device id")
 	}
 
+	// TODO: We're using the team id only to pull the declaration detail. Any reason for us to check
+	// for a host record first in all cases? Normalizing the UDID for capitalization?
```

Comment:
we already use the uuid as it comes from the protocol to enqueue commands, add entries in the db, etc in other places. I think it would be a nice optimization to move this only where it's needed 👍